src/main/java/github/weichware10/toolbox/gui/EndWindowController.java

```diff
@@ -0,0 +1,75 @@
+package github.weichware10.toolbox.gui;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+import github.weichware10.toolbox.Main;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.toolbox.Main' import. Should be before 'java.util.ResourceBundle'.